website/content/docs/concepts/adaptive-overload-protection/index.mdx

```diff
@@ -60,21 +58,24 @@ In some cases, a sudden influx of write requests that exceeds Vault's hardware
 capacity can result in the writes queueing for so long that every request times
 out before the write can make it through the queue. This makes Vault effectively
 unavailable to clients even though it is still processing requests and storing
-data as fast as it can. This is illustrated in the test results shown below for
-a workload of 100% logins.
+data as fast as it can. This can also result in Replication lagging behind causing
+data discrepancies between replicated clusters. This is illustrated in the test
```

Comment:
🤔 I'm not sure this addition works just here. I like it and we could add a note about replication somewhere but for example, "This is illustrated" previously made sense because "this" was referring to the previous sentence's "... makes Vault effectively unavailable..." but now it's a bit confusing because it sounds like the image is illustrating something to do with replication lag?

Reply:
I moved that info under the graph.